visdiff/src/index.js

```diff
@@ -79,6 +79,8 @@ function renderScreenshots (commitRange) {
       console.log(`Error creating work dir clone:`, result.error, result.stderr)
       process.exit(1)
     }
+  } else {
+    console.log(`Note: using existing work dir: ${WORK_DIR}. Try clearing out this directory if you are having problems.`)
```

Comment:
Yeah I was considering that too, it's unnecessarily failure prone. Will open a ticket for it.